### ISSUE TITLE
Error calculating roc_auc

### ISSUE THREAD

**Issue by nanounanue** [dssg/triage#227]
```

15:50:12
WARNING:root:1year

15:50:15
ERROR:root:Child error: Traceback (most recent call last):

15:50:15
File "/usr/local/lib/python3.6/site-packages/triage/experiments/multicore.py", line 276, in test_and_evaluate

15:50:15
example_frequency=split_def['example_frequency']

15:50:15
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 225, in evaluate

15:50:15
labels_sorted.tolist(),

15:50:15
File "/usr/local/lib/python3.6/site-packages/catwalk/evaluation.py", line 145, in _generate_evaluations

15:50:15
parameter_combination

15:50:15
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 46, in __call__

15:50:15
return self.function(*params, **kwparams)

15:50:15
File "/usr/local/lib/python3.6/site-packages/catwalk/metrics.py", line 79, in roc_auc

15:50:15
return metrics.roc_auc_score(labels, predictions_proba)

15:50:15
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/ranking.py", line 260, in roc_auc_score

15:50:15
sample_weight=sample_weight)

15:50:15
File "/usr/local/lib/python3.6/site-packages/sklearn/metrics/base.py", line 81, in _average_binary_score

15:50:15
raise ValueError("{0} format is not supported".format(y_type))

15:50:15
ValueError: continuous format is not supported
```

**Comment by thcrock** [comment]
I believe this is also covered by https://github.com/dssg/catwalk/pull/30 which has to be reopened in this repo.

**Comment by thcrock** [comment]
Fixed in #373
